dataproc_jupyter_plugin/handlers.py

```diff
@@ -119,6 +121,23 @@ def get_cached_credentials():
 
         return credentials
 
+def gcpServiceUrl(cmd):
```

Comment:
This method is wrong in multiple ways:

1. Not setting the stdin to `/dev/null` (which can cause gcloud to hang in some scenarios)
2. Ignoring what to do when the exit code is not 0.
3. Naming a variable `_` and then using that variable.
4. Relying on a gcloud command that is not guaranteed to be stable.
5. Relying on stderr output of said gcloud command that is not meant to be processed by scripts.
6. Using camel-case for a Python method name.

More importantly, though, this is replicating logic we already have in the `google.cloud.jupyter_config` package.

Instead, we should add an import statement above of

```py
from google.cloud.jupyter_config.config import get_gcloud_config
```

... and then change this method to be something like:

```py
def gcp_service_url(service_name, default_url=None):
    default_url = default_url or f'https://{service_name}.googleapis.com/'
    configured_url = get_gcloud_config(f'configuration.properties.api_endpoint_overrides.{service_name}')
    return configured_url or default_url
```

... and the calls to it would be things like:

```py
        dataproc_url = gcp_service_url('dataproc')
        compute_url = gcp_service_url('compute', default_url='https://compute.googleapis.com/compute/v1/')
        metastore_url = gcp_service_url('metastore')
        cloudkms_url = gcp_service_url('cloudkms')
        cloudresourcemanager_url = gcp_service_url('cloudresourcemanager')
        datacatalog_url = gcp_service_url('datacatalog')
        storage_url = gcp_service_url('storage', default_url='https://storage.googleapis.com/storage/v1/')
```